man/org.freedesktop.machine1.xml

```diff
@@ -112,10 +112,12 @@ node /org/freedesktop/machine1 {
                        in  b mkdir);
       CopyFromMachine(in  s name,
                       in  s source,
-                      in  s destination);
+                      in  s destination,
+                      in  b force_copy);
```

Comment:
hmm, so we probably should make this a flags param, i.e. `in t flags`. This smells as if we might want to add more sooner or later.

Flags in D-Bus suck, but it's the easiest still.

Reply:
Also, i think the internal flag should be called "replace", not "force". i.e. "force" could mean a lot, but this is specifically about replacing existing files, hence the naming should communicate that.

Reply:
(i think it's fine if the switch on the cmdline is nonetheless called --force, since cp calls it that way)

Reply:
Thank you for the review! I appreciate it as I am still learning about the repo.

> hmm, so we probably should make this a flags param, i.e. `in t flags`. This smells as if we might want to add more sooner or later.
> 
> Flags in D-Bus suck, but it's the easiest still.

Should we create an enum to store the possible flags here?

Reply:
> Should we create an enum to store the possible flags here?

Yes, create an enum, but it's not exposed in D-Bus itself. The D-Bus type for flags is uint64_t, or "t" in the D-Bus type system. You can then define an enum with all the values for use in the internal logic. See `UnitFileFlags` for an example. It also defines a mask that's used to check that only defined flags are passed, and returns an error if there are undefined flags. This is so that we can add new flags in the future without breaking code that's setting undefined flags. You should do the same for the new enum.